File: NetBeansProjects/java/simulator/src/main/java/com/maicol/simulator/Force.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.maicol.simulator;

/**
 *
 * @author Maicol
 */
  public abstract class Force {
    private final float mass;
    private final float acceleration;

    public Force(float mass, float acceleration) {
        this.mass = mass;
        this.acceleration = acceleration;
    }

    public float getMass() {
        return mass;
    }

    public float getAcceleration() {
        return acceleration;
    }
}

File: php/exercises/3.simulator/php/force.php
<?php
abstract class force {
 
    private float $mass;
    private float $acceleration;

    function __construct(float $mass, float $acceleration) {
        $this->mass = $mass;
        $this->acceleration = $acceleration;
    }
   
    function getMass(): float {
        return $this->mass;
    }

    function getAcceleration(): float {
        return $this->acceleration;
    }
}
?>